apps/vaults/hooks/useSolverPartnerContract.ts

```diff
@@ -85,13 +86,16 @@ export function useSolverPartnerContract(): TSolverContext {
 		txStatusSetter: React.Dispatch<React.SetStateAction<TTxStatus>>,
 		onSuccess: () => Promise<void>
 	): Promise<void> => {
+		const partnerContract = networks[safeChainID]?.partnerContractAddress;
+
 		assert(request.current, 'Request is not set');
 		assert(request.current?.inputToken, 'Input token is not set');
+		assertAddress(partnerContract, 'partnerContract');
```

Comment:
I think we can leave it, but it would have been better having named params `assertAddress({ thingy: partnerContract, name: 'partnerContract'})`